API/src/routes/login.js

```diff
@@ -0,0 +1,81 @@
+var login = require('express').Router();
+var db = require('../db');
+var PQ = require('pg-promise').ParameterizedQuery;
+var bcrypt = require('bcrypt');
+var jwt = require('jsonwebtoken');
+var secret = require('./secret');
+var app = require('express')();
+
+
+
+const expiryTime = 180; //the time for expiry for a token (3 minutes for testing purposes)
```

Comment:
Has this been tested? Can you change this to 1hr now, or something along that time?